wcfsetup/install/files/lib/system/bbcode/BBCodeHandler.class.php

```diff
@@ -191,4 +192,109 @@ public function getImageExternalSourceWhitelist()
 
         return \array_unique($hosts);
     }
+
+    /**
+     * Exports a require.js requirement for the localization of the editor based
+     * on the current locale. Returns an empty string when there is no available
+     * localization or the locale equals the bundled value 'en'.
+     *
+     * @since 6.0
+     */
+    public function getEditorLocalization(): string
+    {
+        $availableTranslations = [
+            'af',
+            'ar',
+            'ast',
+            'az',
+            'bg',
+            'bn',
+            'bs',
+            'ca',
+            'cs',
+            'da',
+            'de-ch',
+            'de',
+            'el',
+            'en-au',
+            'en-gb',
+            'eo',
+            'es-co',
+            'es',
+            'et',
+            'eu',
+            'fa',
+            'fi',
+            'fr',
+            'gl',
+            'gu',
+            'he',
+            'hi',
+            'hr',
+            'hu',
+            'id',
+            'it',
+            'ja',
+            'jv',
+            'kk',
+            'km',
+            'kn',
+            'ko',
+            'ku',
+            'lt',
+            'lv',
+            'ms',
+            'nb',
+            'ne',
+            'nl',
+            'no',
+            'oc',
+            'pl',
+            'pt-br',
+            'pt',
+            'ro',
+            'ru',
+            'si',
+            'sk',
+            'sl',
+            'sq',
+            'sr-latn',
+            'sr',
+            'sv',
+            'th',
+            'tk',
+            'tr',
+            'tt',
+            'ug',
+            'uk',
+            'ur',
+            'uz',
+            'vi',
+            'zh-cn',
+            'zh',
+        ];
+
+        $locale = WCF::getLanguage()->getBcp47();
+        if (\in_array($locale, $availableTranslations, true)) {
+            return \sprintf(
+                '"ckeditor5-translation/%s",',
+                $locale
+            );
+        }
+
+        $index = \strpos($locale, '-');
+        if ($index !== false) {
+            $locale = \substr($locale, 0, $index);
+
+            if (\in_array($locale, $availableTranslations, true)) {
+                return \sprintf(
+                    '"ckeditor5-translation/%s",',
+                    $locale
+                );
+            }
+        }
```

Comment:
```suggestion
        [$languageCode] = \explode('-', $locale, 2);

        if (\in_array($languageCode, $availableTranslations, true)) {
            return \sprintf(
                '"ckeditor5-translation/%s",',
                $languageCode
            );
        }
```

This will perform a redundant `in_array` check for unknown locales without region code, but is much simpler to understand.